DscResourceTestHelper/TestHelper.psm1

```diff
@@ -319,7 +319,7 @@ function Initialize-TestEnvironment
     } 
     
     # The folder where this module is found
-    [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path))
+    [String] $moduleRoot = Split-Path -Parent (Split-Path -Parent (Split-Path -Parent (Split-Path -Parent $Script:MyInvocation.MyCommand.Path)))
```

Comment:
So it's 
github: xWebAdmin\Tests\DscResource.Tests\DscResourceTestHelper\TestHelper.psm1
gallery: xWebAdmin\Tests\0.2.0.0\DscResourceTestHelper\TestHelper.psm1

so both have one extra layer